lib/classifier-reborn/extensions/hasher.rb

```diff
@@ -18,22 +18,22 @@ def without_punctuation(str)
 
     # Return a Hash of strings => ints. Each word in the string is stemmed,
     # interned, and indexes to its frequency in the document.
-    def word_hash(str)
-      word_hash   = clean_word_hash(str)
+    def word_hash(str, language='en')
```

Comment:
Nit: we follow the [GitHub Ruby Styleguide](https://github.com/styleguide/ruby) for new code. It states:

> Use spaces around the = operator when assigning default values to method parameters.

Would you mind updating your changes to match this?